### DISCUSSION TITLE
possible issue with spaces between assigners.

### DISCUSSION THREAD

**Discussion by lelandstarr** [modelint/scrall#7] [General]
Maybe I am missing something, but I ran into this when running a simple unit test.

test that
x=y a=b
should I believe result as two assigners one that assigns the value of y to the field x and another that assigns the value of b to field a.
however the PEG captures "y a" as an attribute and then trips up on the following = sign.
a tab or double space fixes this.

**Comment by lelandstarr** [comment]
I took this from an example early in the wiki.  looks like the example was misleading, as the solution was later identified.
non issue.

**Reply by modelint** [reply]
Please send me a link to the example if you trip on it again and I will update it

**Reply by lelandstarr** [reply]
https://github.com/modelint/scrall/wiki/Action-semantics

Here is an example of an arbitrary computation sequence:

y = scale(x) z = filter(i1, i2, ... in) result = y + z

I guess technically this wasn't example code so much as a logical example that I may have mis interperated.
